### PR TITLE
ERC-1363: Move to Last Call

### PR DESCRIPTION
**Pull request by vittominacori**
Moves ERC-1363 to Final.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1363 state from Draft to Last Call

**Comment by axic** [comment]
It is not possible to move from draft to final in a single step, please refer to EIP-1.

**Comment by vittominacori** [comment]
Updated to Last Call.

It was Last Call first but automerger fails. So I moved to final.

File: EIPS/eip-1363.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 1363
 title: ERC-1363 Payable Token
 author: Vittorio Minacori (@vittominacori)
 discussions-to: https://github.com/ethereum/eips/issues/1363
-status: Draft
+status: Last Call
```
Comment (MicahZoltu) [inline EIPS/eip-1363.md:6]:
```suggestion
status: Last Call
review-period-end: 2020-09-20
```

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Last Call must have a `review-period-end` of at least 2 weeks in the future.

**Comment by MicahZoltu** [comment]
`## Security Considerations` section is required for all EIPs.  It comes right before the Copyright section.  If you believe there are no security considerations, the section can just include one-sentence saying as much.

**Comment by MicahZoltu** [comment]
I would also recommend changing:
```
Every Payable Token compliant contract MUST implement the ERC-1363 interface other than ERC-20 and ERC-165 interfaces.
```
to
```
Every Payable Token compliant contract **MUST** implement the ERC-1363 interface as well as the ERC-20 and ERC-165 interfaces.
```

**Comment by MicahZoltu** [comment]
I recommend using the RFC 2119 style for `**MUST**` rather than `MUST`.

**Comment by MicahZoltu** [comment]
Recommend changing:
```
Description of a Payable Token compatible with the ERC-20 definition. Also description of a Token Receiver and/or Spender that accepts Payable Token payments.
```
to
```
Defines a token interface for ERC-20 tokens that supports executing recipient code on transfer.
```

**Comment by MicahZoltu** [comment]
Recommend changing the abstract to be a bit more descriptive, maybe mentioning the highlights of the new functions.  I generally recommend the abstract contain a human-readable and terse version of the specification.  Something like, "Adds a `transferAndCall` method that allows the caller to specify that the that the recipient contract will be called with calldata supplied by the caller. ...".  Maybe add a sentence or two about some other features, but I recommend leaving out the specific details like all of the overloads, and the exact mechanism for callbacks, etc.

Basically, state the core protocol in human readable words, but don't cover all of the edge cases and little details.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
The Security Considerations section is a hard requirement.  The rest are feedback that I encourage applying, but aren't strictly required to move to Last Call.

**Comment by MicahZoltu** [comment]
I know I said that the other changes were optional, but on doing a final review I think this change is necessary afterall because the current wording doesn't align with the rest of the document:
> Every Payable Token compliant contract **MUST** implement the ERC-1363 interface other than ERC-20 and ERC-165 interfaces.

to
> Every Payable Token compliant contract **MUST** implement the ERC-1363 interface as well as the ERC-20 and ERC-165 interfaces.

"other than" implies that someone implementing this standard **MUST NOT** implement ERC-20 and ERC-165, which I don't think was the intent.

Along with that change, please also set the review period end to 2 weeks from the date of that change as I *hope* this is the last change before Last Call.  😄

**Review by MicahZoltu — APPROVED** [review]
